src/samples/flow/am2315/Kconfig

```diff
@@ -0,0 +1,4 @@
+config FLOW_AM2315_SAMPLE
+	bool
+	depends on FLOW_FBP_GENERATOR_SAMPLES && FLOW_NODE_TYPE_AM2315
```

Comment:
Don't you have to depend on all node types used on the sample? I know that some samples do that and other don't, but we need consistency.